### PR TITLE
Add videown.md

### PR DESCRIPTION
**Pull request by lost-inwind**
Grant Application: VIDEOWN

VIDEOWN is a decentralized video NFT platform.

### CODE REVIEW

**Comment by SotaWatanabe** [comment]
Thank you for your application. The purpose of this grant program is not to fund a specific project in a particular chain but to fund a general-purpose ink! smart contracts on potentially all parachains. In this sense, this application is out of scope.

**Comment by lost-inwind** [comment]
Hi @SotaWatanabe . This application can be built on any Wasm-supporting parachain. Intended to demonstrate the use of ink! smart contract to realize the nft market. It may have been misleading to use a particular chain as an example earlier.

**Review by 0xMarkian — CHANGES_REQUESTED** [review]
Hey, thanks for the comment @lost-inwind! Can you indicate how you think this application will be useful for the ecosystem? Is this an infrastructure or a business case? We are willing to fund the code that will be used either in other people's programs or by itself as a commercial business deployed on-chain constantly running transactions.

File: applications/videown.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+- **FTE:**  2
+- **Costs:** 18,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0 |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how use the product, display and explain the function of each component. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 1. | Smart contract: videown-nft | We will deliver a ink! smart contract that will realize the function of NFT market and comply with [psp-34](https://github.com/w3f/PSPs/blob/master/PSPs/psp-34.md). |
```
Comment (DoubleOTheven) [inline applications/videown.md:145]: Would you please provide more details on what you will build with ink!? A budget of $18k does not make sense for a single NFT contract. ink!ubator is meant to fund ink!-related projects, not APIs and infra that are unrelated.

I would like to see more interesting features on the contract side. Please elaborate on what the contract/s would do.

Comment (lost-inwind) [reply applications/videown.md]: Thank you for the feedback @DoubleOTheven . The contract will contain three types of functions. Not only the basic functions of NFT, but also NTF market functions and Economic model.

**Comment by lost-inwind** [comment]
> Hey, thanks for the comment @lost-inwind! Can you indicate how you think this application will be useful for the ecosystem? Is this an infrastructure or a business case? We are willing to fund the code that will be used either in other people's programs or by itself as a commercial business deployed on-chain constantly running transactions.

Hey @0xMarkian , thanks for your reminder. We have noticed that the category of Business case has been added. We think this application might be more in line with the Business case category. We have added an Ink! Ecosystem Impact section to the proposal.

**Comment by TtomaS7** [comment]
@lost-inwind 
Thank you for expressing interest in ink!ubator and for your commitment to the innovation in the ink! ecosystem.

We appreciate the time and effort you invested in applying for a grant. Your application has been under review for more than three weeks, reflecting our thorough evaluation process.

However, after meticulous deliberation, we regret to inform you that your application has not been successful on this occasion. This decision was reached based on various elements, including the alignment of your project with our program's current objectives.

We encourage you to continue refining your ideas and hope to see future applications from you.
Thank you once again for your interest and for contributing to the growth of the ink! ecosystem.
